pkg/logs/config/config.go

```diff
@@ -29,6 +30,17 @@ func DefaultSources() []*LogSource {
 		})
 		sources = append(sources, source)
 	}
+
+	if LogsAgent.GetBool("logs_config.container_collect_all") {
+		// append a new source to collect all logs from all containers
+		source := NewLogSource("container_collect_all", &LogsConfig{
```

Comment:
Why do we add a Docker source ? When starting collecting logs on k8 running on containerd, we will have something like this 

```
kube-system/kube-controller-manager/kube-controller-manager
  -----------------------------------------------------------
    Type: file
    Path: /var/log/pods/8d989b65-ba83-11e8-b67f-025f4577d1f5/kube-controller-manager/*.log
    Status: OK
    Inputs: /var/log/pods/8d989b65-ba83-11e8-b67f-025f4577d1f5/kube-controller-manager/0.log

  container_collect_all
  ---------------------
    Type: docker
    Status: Pending
```

Reply:
Agreed, it's a corner case, I don't have a good solution to address this for now. Any idea ?

Reply:
Not for now